boards/arm/particle_boron/particle_boron.dts

```diff
@@ -29,8 +29,8 @@
 
 	tx-pin = <37>;
 	rx-pin = <36>;
-	rts-pin = <39>;
-	cts-pin = <38>;
+	rts-pin = <38>;
+	cts-pin = <39>;
```

Comment:
Well spotted, I double checked the diagrams and you're right for both versions

Reply:
Sorry, but appears it was right in the first place for both modems.

Please revert changes.

Billy..

![image](https://user-images.githubusercontent.com/43581311/58506218-b7c12500-8186-11e9-986a-ed26717fbdc7.png)

Reply:
The schematic shows that P1.07/RTS_UB_NRF goes, via the buffer, directly to RTS_UB on the modem and P1.06/CTS_UB_NRF goes to CTS_UB on the modem. Normally one hooks up the UART's RTS pin to the modem's CTS and the UART's CTS pin to the modem's RTS. They should be swapped around, just like the UART's RX goes to the modem's TX. I could be wrong but please look again; I got hangs in the UART driver before I swapped them.

![Boron u-blox schematic](https://docs.particle.io/assets/images/boron/schematic-ublox.png)